README.md

```diff
@@ -110,6 +120,19 @@ Note: when storing points, the following conventions are used:
    default, the adapter stores all numeric fields as floats. This can be changed
    by enumerating integer fields via `intFields` option.
 
+### Optimizations
```

Comment:
I'm sitting on a fence here - should this be part of the README? On one hand, user won't care about a technical detail like optimization, she cares about speed - the query should be always fast and if it isn't, she will file a bug without actually caring why. So this would belong into technical documentation, rather than in the README. On the other hand, the adapters don't have any sort of technical/developer docs yet, so README is probably fine for now.